apps/zotonic_mod_backup/src/support/z_file_crypto.erl

```diff
@@ -0,0 +1,175 @@
+%% @author Maas-Maarten Zeeman <maas@channel.me>
+%% @copyright 2024 Maas-Maarten Zeeman
+%% @doc Decrypt and encrypt files.
+
```

Comment:
Add an `@end` here, as otherwise the APL text is added to the `@doc`.

Reply:
Maybe also rename the file to `backup_file_crypto.erl`  (or us it something that would be nice for other modules to use? then we could move it to stdlib or zotonic-support)

Reply:
Going to rename this.